docs/docs/faq.md

```diff
@@ -5,72 +5,29 @@
 Nebari regenerates this file on every run. This means it will be removed by the operating system during its cleanup process,
 but running the `nebari deploy` command again as a Nebari administrator will update/create a `NEBARI_KUBECONFIG` file for you.
 
-## How are Nebari conda user environments created? Who creates them?
+## How are conda user environments created? Who creates them?
 
-The short answer: there are currently _two_ ways of creating environments, as we are in the process of migrating Nebari to conda-store,
-and so which way depends on your use-case.
+`Conda-store` manages all environments on the Nebari. It allows users to create private environments in their own namespace, or shared environments under a group namespace. For more details check out the doc on [creating environments in Nebari via conda-store](/docs/tutorials/creating-new-environments).
 
-The longer answer:
+Additionally, there is a legacy approach which is still available in Nebari. Administrators can create global environments by specifying the environment in `nebari-config.yml`. Environments specified in this way will be made available for all users and services under the `nebari-git` namespace on `conda-store`.
 
-- For global environments, you can specify the environment in `nebari-config.yml`, and it will be made available for all users and services.
-- By comparison, creating the environments through conda-store will provide more granular control over certain settings and permissions.
-
-As Nebari and conda-store mature, the intent is to migrate exclusively to conda-store for environment creation and management.
-
-## What if I need to install package `X`, and it's not available in the environment?
-
-You can add the package to the `nebari_config.yml`. If you don't have access to the deployment repo,
-you'll need to contact your Nebari administrator to include the required package.
-
-## What's included in the conda environment if I want to use Dask?
-
-<!-- TODO: will need to update the conda-feedstock -->
+## What should be included in the environment if I want to use Dask?
 
 There are drop-in replacements for `distributed`, `dask`, and `dask-gateway` with the correct pinned versions available via the [Nebari Dask metapackage](https://github.com/conda-forge/nebari-dask-feedstock). Example: `nebari-dask==||nebari_VERSION||`.
 
-## What packages are needed in your environment to create a dashboard?
-
-When deploying an app with JHub App Launcher, you need to have the following in your environment:
-
-- `jhub-apps` package
-- packages corresponding to the dashboard framework (for example, `panel`, `gradio`, etc.)
-- any other libraries required for the analysis in the dashboard creation script/notebook
-
```

Comment:
Why delete this?  Is it covered elsewhere?

Reply:
the jhub-apps package is actually not required but also, there are now dedicated pages for dashboarding - [this tutorial](https://www.nebari.dev/docs/tutorials/create-dashboard#deploy-the-dashboard) and [this how-to](https://www.nebari.dev/docs/how-tos/jhub-app-launcher)